src/index.js

```diff
@@ -48,16 +48,23 @@ export default async (
       .filter(({ shouldTransform }) => shouldTransform(urlString))
       .forEach(({ getHTML }) => {
         transformations.push(async () => {
-          let html = await cache.get(urlString);
+          try {
+            let html = await cache.get(urlString);
 
-          if (!html) {
-            html = await getHTML(urlString);
-            await cache.set(urlString, html);
-          }
+            if (!html) {
+              html = await getHTML(urlString);
+              await cache.set(urlString, html);
+            }
 
-          node.type = `html`;
-          node.value = html;
-          node.children = undefined;
+            node.type = `html`;
+            node.value = html;
+            node.children = undefined;
+          } catch (error) {
+            // eslint-disable-next-line no-console
+            console.error(`The following error appeared while processing: ${urlString}`);
+            
+            throw error
```

Comment:
This will result in 2 separate errors in output (each with "ERROR" badge).

Ideally we throw single error here, how about this: 
```suggestion
            error.message = `The following error appeared while processing: ${urlString}\n\n${error.message}`
            throw error
```
(pretty low-fi, might be nicer with having custom error classes etc - like in https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error#ES6_Custom_Error_Class)

Reply:
We can do a custom error in another PR just so we don't pollute this PR 🙂